mqgo/src/meqa/mqswag/parser.go

```diff
@@ -167,6 +167,20 @@ func CreateSwaggerFromURL(path string, meqaPath string) (*Swagger, error) {
 	return (*Swagger)(specDoc.Spec()), nil
 }
 
+func GetWhitelistSuites(path string) (map[string]bool, error) {
+	whitelistBytes, err := ioutil.ReadFile(path)
+	if err != nil {
+		mqutil.Logger.Printf("can't read file %s", path)
```

Comment:
So we are printing the error here as well and in the main method as well. We can do it a single place.